src/i18n/cs/plugins/exec/sim-host-dialogs.html

```diff
@@ -4,7 +4,7 @@
 <cordova-dialog id="exec-dialog" caption="Neošetřené volání metody Exec" data-loc-id="f7283850">
     <cordova-group>
         <p data-loc-id="a1a63f26">Pro následující volání metody Exec není k dispozici obslužná rutina:</p>
-        <p style="font-style:italic"><span id="exec-service" data-loc-id="ba2b4c80">služba</span>.<span id="exec-action" data-loc-id="e4c1ba28">akce</span></p>
+        <p style="font-style:italic"><span id="exec-service" data-loc-id="ba2b4c80">služba</span>.<span id="exec-action" data-loc-id="e4c1ba28">akce</span>(<span id="exec-args" data-loc-id="veyymo5t">arguments</span>)</p>
```

Comment:
No, I changed files manually). Should I revert changes in both `src/i18n` and `tools/i18n` folders?